_documentation/faq.md

```diff
@@ -145,3 +145,13 @@ TTreeReader tree_reader("hepmc3_tree", file);
 tree->Scan("hepmc3_event.attribute_string", "hepmc3_event.attribute_id==0 && hepmc3_event.attribute_name ==\"GenCrossSection\"", "colsize=30", 1, tree->GetEntries()-1);
 ```
 The “1” at the end of the file name indicates that these events are used in work/eic2/EPIC/RECO/24.06.0/epic_craterlake/DIS/NC/18x275/minQ2=1/pythia8NCDIS_18x275_minQ2=1_beamEffects_xAngle=-0.025_hiDiv_1.xxxx.hepmc3.tree.root
+
+**Q: How do I make new python packages available in eic-shell container environment?** 
+
+A user might need to add a new python package or update the version of a package available in the eic-shell container. The following steps would be needed:
```

Comment:
Limit the statement to just new packages since that's what the instructions are focused on. Updated versions are easier (even if steps 3 and 4 are similar). Maybe add a next Q on that.